web/packages/teleterm/src/ui/AccessRequestCheckout/useAccessRequestCheckout.test.tsx

```diff
@@ -64,3 +67,120 @@ test('fetching requestable roles for servers uses UUID, not hostname', async ()
     })
   );
 });
+
+test('fetching requestable roles kube_cluster resource without namespace request', async () => {
```

Comment:
I think it's more readable:
```suggestion
test('fetching requestable roles for a kube_cluster resource without specifying a namespace', async () => {
```